deploy/releases/0.8.1/readme.adoc

```diff
@@ -0,0 +1,34 @@
+= Runtime Component Operator v0.8.1
+
+== Changelog
+
+All notable changes are documented in the link:++https://github.com/application-stacks/runtime-component-operator/blob/main/CHANGELOG.md#080++[Changelog].
+
+== Supported Environments
+
+The Runtime Component Operator is tested and built on the following CPU architectures:
+
+* amd64
```

Comment:
@halim-lee We can list p and z architectures for the new release and mention the install options

* amd64 (Supported with Installation options 1, 2 and 3)
* ppc64le (IBM Power. Supported with Installation options 2 and 3 only)
* s390x (IBM Z Linux. Supported with Installation options 2 and 3 only)